weather.py

```diff
@@ -197,19 +213,17 @@ def __init__(self):
                 try:
                     pygame.display.init()
                 except pygame.error:
-                    print('Driver: {0} failed.'.format(driver))
-                    syslog.syslog('Driver: {0} failed.'.format(driver))
+                    self.log.debug("Driver: {driver} failed.")
                     continue
                 found = True
                 break
 
             if not found:
-                raise Exception('No suitable video driver found!')
+                self.log.exception("No suitable video driver found!")
```

Comment:
This is a fatal exception that should cause the program to exit. Does this logging setup account for that?

Reply:
No, it doesn't. As it stands now, the service file will continue to restart the script every 5 seconds either way. I think it should be fairly easy to handle this exception in the `start=stop` PR, though. We should be able to simply log the error and set `running` to `False`